### ISSUE TITLE
Contributors Need a Way To Run The Tests

### ISSUE THREAD

**Issue by jamesonwilliams** [aws-amplify/amplify-android#301]
Internally, AWS maintainers run integration tests against some pre-allocated resources.

Configuration files that point to these resources can be attained by running `.circleci/copy-configs`. As the directory name implies, this script is run as part of the CircleCI process when a PR is proposed to the main repo.

But, it is only accessible by Amazon employees. And there is no documentation about how to re-create them, if you don't have access. Nor can a contributor even inspect their structure.

TLDR: we need a simple mechanism for contributors to run the integration tests.

**Comment by jpignata** [comment]
It would be ideal if we supplied a CloudFormation template that created a stack with randomly named resources that were used during testing with explicit steps for a developer to setup and teardown these resources.

**Comment by jamesonwilliams** [comment]
@jpignata Agreed, that's one very attractive option. We could also use the Amplify CLI, directly (it wraps CloudFormation.) But, it's missing some of the "undo" operations.

Independent of those, I'm fond of putting up a public endpoint that contributors can use to run tests against. There are some challenges around that, like throttling and security, but it would be convenient for community members trying to onboard.
